standalone/test/unit_tests/api/unittest_io_state.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2019-2021 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#include <gtest/gtest.h>
+
+#include "psen_scan_v2_standalone/io_state.h"
+
+namespace psen_scan_v2_standalone_test
+{
+using psen_scan_v2_standalone::IOState;
+using psen_scan_v2_standalone::PinState;
+
+static const PinState PIN_STATE(0, "", false);
```

Comment:
I dont get the idea here...

Reply:
I think it is to avoid duplication.

Reply:
Not sure if a agree here. It hides details of the expectation. Reading this also showed that this test seems to lack some variance of its cases. @martiniil wdyt?